resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/SparkKubernetesClientFactory.scala

```diff
@@ -18,6 +18,8 @@ package org.apache.spark.deploy.k8s
 
 import java.io.File
 
+import scala.language.implicitConversions
```

Comment:
Is this required globally in this file? Otherwise, can we move this into `object ClientType` like `import Config._`?